wrappers/python/README.md

```diff
@@ -18,15 +18,33 @@ Note: To install via `setup.py`, you need a suitable [build environment](https:/
 
 ## Usage
 
+An example for writing a code to a PNG file with the help of PIL is:
+
+```python
+import sys
+import zxingcpp
+from PIL import Image
+
+if len(sys.argv) < 3:
+    img = zxingcpp.write_barcode(zxingcpp.BarcodeFormat.QRCode,
+                                 'I have the best words.', width=200, height=200)
+else:
+    img = zxingcpp.zxingcwrite_barcode(zxingcpp.barcode_format_from_str(sys.argv[1]),
+                                       sys.argv[2], width=200, height=200)
+Image.fromarray(img).save('test.png')
+```
+
+An example for reading a code from a PNG file with the help of OpenCV is:
+
 ```python
-import cv2
+import sys
 import zxingcpp
+from cv2 import imread
 
-img = cv2.imread('myimage.png')
+img = imread(sys.argv[1])
```

Comment:
Please keep the module name here as well. For this type of "show someone at a glance what it going on" purpose I believe that helps in understanding the code more quickly. I also think using PIL here is maybe even more "basic" but thought that a comment would be useful:
```
img = Image.open(sys.argv[1])   # or using OpenCV: cv2.imread(sys.argv[1])
```